File: app/src/main/java/co/garmax/materialflashlight/features/modes/TorchMode.java
package co.garmax.materialflashlight.features.modes;

/**
 * Just steady light
 */
public class TorchMode extends ModeBase {

    @Override
    public void start() {
        setBrightness(MAX_LIGHT_VOLUME);
    }

    @Override
    public void stop() {
        setBrightness(MIN_LIGHT_VOLUME);
    }

    @Override
    public boolean checkPermissions() {
        return true;
    }
}

File: app/src/main/java/co/garmax/materialflashlight/features/modes/TorchMode.kt
package co.garmax.materialflashlight.features.modes

/**
 * Just steady light
 */
class TorchMode : ModeBase() {

    override fun start() {
        setBrightness(MAX_LIGHT_VOLUME)
    }

    override fun stop() {
        setBrightness(MIN_LIGHT_VOLUME)
    }

    override fun checkPermissions(): Boolean {
        return true
    }
}